xtask/src/ci/qemu.rs

```diff
@@ -279,6 +283,36 @@ fn test_testudp() -> Result<()> {
 	let socket = UdpSocket::bind("127.0.0.1:0")?;
 	socket.connect("127.0.0.1:9975")?;
 	socket.send(buf.as_bytes())?;
+
+	let mut buf = [0; 128];
+	let received = socket.recv(&mut buf)?;
+	eprintln!("[CI] receive: {}", from_utf8(&buf[..received])?);
+
+	Ok(())
+}
+
+fn test_miotcp() -> Result<()> {
+	thread::sleep(Duration::from_secs(10));
+	let buf = "exit";
+	eprintln!("[CI] send {buf:?} via TCP to 127.0.0.1:9975");
+	let mut stream = TcpStream::connect("127.0.0.1:9975")?;
+	stream.write_all(buf.as_bytes())?;
+
+	let mut buf = vec![];
+	let received = stream.read_to_end(&mut buf)?;
+	eprintln!("[CI] receive: {}", from_utf8(&buf[..received])?);
+
+	Ok(())
+}
+
+fn test_mioudp() -> Result<()> {
```

Comment:
You added `socket.recv` to `test_testudp` but not to `test_mioudp`. Is that on purpose?

Reply:
I forgot it and fixed it.